EDDiscovery/Forms/DownloadManagerForm.Designer.cs

```diff
@@ -17,6 +17,7 @@ protected override void Dispose(bool disposing)
             {
                 components.Dispose();
             }
+            font?.Dispose();
```

Comment:
I've never seen the designer re-write anything outside of the commented region that tells you not to modify that which is contained inside the commented region. Dispose is safe in that regard, as designer intentionally uses components to avoid modifying dispose after is created from boilerplate.

As for the second paragraph... You are correct. I was playing around with this in a code base that was modified to dispose of this form via a using block, and I didn't even think about the repercussions of not also including that change.